var/spack/repos/builtin/packages/fujitsu-mpi/package.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class FujitsuMpi(Package):
+    """Fujitsu MPI implementation only for Fujitsu compiler."""
+
+    homepage = "https://www.fujitsu.com/us/"
+
+    version('3.0')
+
+    provides('mpi@3.1:')
+
+    def install(self, spec, prefix):
+        raise InstallError(
+            'Fujitsu MPI is not installable; it is vendor supplied')
+
+    def setup_dependent_package(self, module, dependent_spec):
+        if '%fj' in dependent_spec:
+            self.spec.mpicc = join_path(self.prefix.bin, 'mpifcc')
```

Comment:
```suggestion
            self.spec.mpicc = self.prefix.bin.mpifcc
```
Same below

Reply:
Thanks for comment and fix! I felt that this fix was simpler and better. I fixed it on the latest commit!